src/scenes/AcceleratorRouter/Cohorts/CohortsListView.tsx

```diff
@@ -96,42 +118,46 @@ const CohortsTable = ({ filterModifiers, extraTableFilters }: CohortTableProps)
       fuzzySearchColumns={fuzzySearchColumns}
       id='cohortsTable'
       Renderer={CohortCellRenderer}
+      rightComponent={actionMenus}
       rows={cohorts || []}
+      title={strings.COHORTS}
```

Comment:
Nice, thanks. You could optionally use the `busy= prop` if your selector has a status field (if cohorts request uses an async thunks). Otherwise no worries.